packages/client/src/svelte/main.ts

```diff
@@ -1,59 +1,52 @@
-import "./app.css"
-import App from "./App.svelte"
-import { mount as mountAccountKit } from "@latticexyz/account-kit/bundle"
-import { createConfig } from "@wagmi/core"
-import { getNetworkConfig } from "@mud/getNetworkConfig"
-import { ENVIRONMENT } from "@mud/enums"
-import { supportedChains } from "@mud/supportedChains"
-import { MUDChain } from "@latticexyz/common/chains"
-import { transportObserver } from "@latticexyz/common"
-import { fallback, http, webSocket } from "viem"
-import { mainnet, holesky } from "viem/chains"
+import "./app.css";
+import App from "./App.svelte";
+import { mount as mountAccountKit } from "@latticexyz/account-kit/bundle";
+import { createConfig, http } from "@wagmi/core";
+import { getNetworkConfig } from "@mud/getNetworkConfig";
+import { ENVIRONMENT } from "@mud/enums";
+import { supportedChains } from "@mud/supportedChains";
+import { transportObserver } from "@latticexyz/common";
+import { fallback, webSocket } from "viem";
 
 const getEnvironment = () => {
   switch (window.location.hostname) {
     case "thiscursedmachine.fun":
-      return ENVIRONMENT.REDSTONE
+      return ENVIRONMENT.REDSTONE;
     case "redstone-test.thiscursedmachine.fun":
-      return ENVIRONMENT.REDSTONE_TEST
+      return ENVIRONMENT.REDSTONE_TEST;
     case "garnet.thiscursedmachine.fun":
-      return ENVIRONMENT.GARNET
+      return ENVIRONMENT.GARNET;
     case "garnet-account-kit.thiscursedmachine.fun":
-      return ENVIRONMENT.GARNET_ACCOUNT_KIT
+      return ENVIRONMENT.GARNET_ACCOUNT_KIT;
     default:
-      const urlParams = new URLSearchParams(window.location.search)
+      const urlParams = new URLSearchParams(window.location.search);
       if (urlParams.has("useAccountKit")) {
-        return ENVIRONMENT.DEVELOPMENT_ACCOUNT_KIT
+        return ENVIRONMENT.DEVELOPMENT_ACCOUNT_KIT;
       } else {
-        return ENVIRONMENT.DEVELOPMENT
+        return ENVIRONMENT.DEVELOPMENT;
       }
   }
-}
+};
 
-const environment = getEnvironment()
+const environment = getEnvironment();
 
-const networkConfig = getNetworkConfig(environment)
+const networkConfig = getNetworkConfig(environment);
 
 const wagmiConfig = createConfig({
   // chains: [networkConfig.chain as Chain],
-  chains: [...supportedChains, mainnet, holesky] as MUDChain[],
+  chains: supportedChains,
```

Comment:
I moved these into `supportedChains` to fix the transport issue below (which used `supportedChains` and didn't account for mainnet or holesky)